tests/test_models.py

```diff
@@ -1662,7 +1662,7 @@ def test_get_descendants_recursive_cyclic(self):
         point1.add_pointer(point2, auth=self.consolidate_auth)
         point2.add_pointer(point1, auth=self.consolidate_auth)
 
-        descendants = point1.get_descendants_recursive()
+        descendants = [point1.get_descendants_recursive()]
```

Comment:
This is incorrect; the assertion will always pass because you are building a list with a single item in it (a generator object). Correct fix is on https://github.com/CenterForOpenScience/osf.io/pull/2646